modules/generator/processor/spanmetrics/spanmetrics.go

```diff
@@ -125,10 +134,12 @@ func (p *Processor) aggregateMetricsForSpan(svcName string, rs *v1.Resource, spa
 
 	p.spanMetricsCallsTotal.Inc(registryLabelValues, 1*spanMultiplier)
 	p.spanMetricsSizeTotal.Inc(registryLabelValues, float64(span.Size())*spanMultiplier)
+	p.spanMetricsTargetInfo.Inc(registryLabelValues, 0)
```

Comment:
It shouldn't use the same labels as the rest. `target_info` should contain all resource attributes.

Reply:
ok just for clarity, we want _all_ attributes including the ones that the users are not asking for? Isn't that very very high cardinality?

Reply:
Sorry, I'm not sure. That was what I thought, but it's possible I'm very wrong, so I'll just step back and let the owners clarify this.

Reply:
@gouthamve ^^

Reply:
> Isn't that very very high cardinality?

The cardinality is driven by `job` and `instance`. Adding additional resource labels won't add extra cardinality as each (`job`, `instance`) combination is unique. There won't be two sets of resource labels with the same job, instance but different resource labels.